client/packages/system/src/Encounter/DetailView/DetailView.tsx

```diff
@@ -75,9 +76,20 @@ export const DetailView: FC = () => {
   useEffect(() => {
     if (encounter)
       setSuffix(
-        `${
-          encounter.document.documentRegistry?.name
-        } - ${dateFormat.localisedDateTime(encounter.startDatetime)}`
+        <>
+          <Breadcrumb
+            to={RouteBuilder.create(AppRoute.Dispensary)
+              .addPart(AppRoute.Patients)
+              .addPart(encounter.patient.id)
+              .build()}
+            key={'patient'}
+          >
+            {`${encounter.patient.firstName} ${encounter.patient.lastName}`}
+          </Breadcrumb>
+          <span key={'encounter'}>{` / ${
+            encounter.document.documentRegistry?.name
+          } - ${dateFormat.localisedDate(encounter.startDatetime)}`}</span>
```

Comment:
Another small change -- I removed the "time" component of the Encounter breadcrumb as that seemed a bit unnecessarily verbose for what it is.

Reply:
Related note, find it annoying that large bubble modals from the categories hide the encounter date from the breadcrumbs because I sometimes like to know which encounter I am in... not sure how to best solve it though, maybe another header in the modal or just make the modal smaller or put it somewhere else?